### DISCUSSION TITLE
When will .NET MAUI templates be available in Visual Studio 2022 release?

### DISCUSSION THREAD

**Discussion by zapcity** [dotnet/maui#7561] [Q&A] [answered]
Hi, I recall being on VS 2022 17.0 and they said that to use .NET MAUI templates in VS you need 17.1 (Preview). Then when 17.1 was finally released, the templates weren't there and was told you need to use 17.2 (Preview) to use the VS templates. Then when 17.2 was released the templates still weren't included, and was told you can only access the templates in 17.3 (Preview).

When will the templates will be available in the release version of VS 2022? Or can they be installed from somewhere. Not sure what the current state of the situation is. Thanks.

**Comment by xtuzy** [comment, marked as answer]
yes, it just at preview version, 17.2 preview , 17.3 preview, when i install 17.2 release, i 'm happy to hope maui can run, but i find it say it need 17.3 if want use release version.🤣

**Reply by dansiegel** [reply]
The SDK is "GA" which in reality means nothing because the tooling is still in preview. That's why you only see MAUI support in the preview versions of Visual Studio. That not likely to change until the tooling for MAUI also goes GA.

**Reply by zapcity** [reply]
Thanks for that. Great answer - really helped.

**Comment by vassiliospallis** [comment]
MS doesn't care.

**Reply by zapcity** [reply]
Agree, that's why I started using Flutter.
